Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
> because the Linux platform isn't using PCLs here.

Oh...  well is that why @DDReaper is putting `<Reference Include="MonoGame.Framework.Core" />` into platform specific sections?